prime-router/src/test/kotlin/fhirengine/azure/FhirFunctionTests.kt

```diff
@@ -244,12 +244,13 @@ class FhirFunctionTests {
             emptyMap(),
             emptyList()
         )
-        val message = FhirRouteQueueMessage(
+        val message = FhirTranslateQueueMessage(
```

Comment:
What's the story here?

Reply:
the test logic wasn't testing what it was supposed to be testing